Libraries/Text/TextInput/Multiline/RCTMultilineTextInputViewManager.m

```diff
@@ -22,4 +23,14 @@ - (RCTUIView *)view // TODO(macOS ISS#3536887)
 RCT_REMAP_NOT_OSX_VIEW_PROPERTY(dataDetectorTypes, backedTextInputView.dataDetectorTypes, UIDataDetectorTypes) // TODO(macOS GH#774)
 RCT_REMAP_OSX_VIEW_PROPERTY(dataDetectorTypes, backedTextInputView.enabledTextCheckingTypes, NSTextCheckingTypes) // TODO(macOS GH#774)
 
+#if TARGET_OS_OSX // [TODO(macOS GH#774)
+RCT_CUSTOM_VIEW_PROPERTY(pastedTypes, NSArray<NSPasteboardType>*, RCTUITextView)
+{
+  NSArray<NSPasteboardType> *types = json ? [RCTConvert NSPasteboardTypeArray:json] : nil;
+  if ([view respondsToSelector:@selector(setReadablePasteBoardTypes:)]) {
+    [view setReadablePasteBoardTypes: types];
+  }
+}
+#endif // ]TODO(macOS GH#774)
```

Comment:
Re my earlier comment about a property, this wouldn't need to be a `CUSTOM_VIEW_PROPERTY` if we had a property on the base class?

Reply:
Honestly, I was just copying/following 

https://github.com/microsoft/react-native-macos/blob/main/React/Views/RCTViewManager.m#L419-L427

I assumed this style was chosen as we get a JSON strings stored in a NSString* but want to convert it into a NSArray<NSPasteboardType>